### ISSUE TITLE
Search: make the hotkey to trigger the modal configuration

### ISSUE THREAD

**Issue by humitos** [readthedocs/readthedocs-client#43]
There is an ongoing conversation at https://github.com/readthedocs/readthedocs-sphinx-search/issues/87 that we can continue here probably. There are some sites that use the following hotkeys for search:

* `Ctrl + K`
* `/`
* `Ctrl + F`

Stripe, for example, allows the user to "disable triggering with Ctrl+F".

I think we could define a good default hotkey, but also let the author of the project to configure it as they want.

**Comment by humitos** [comment]
This was migrated to the hotkey addons. Now, the question that comes to my mind is "Do we still want to have a `trigger-keycode=` and `trigger-selector=` as part of the WebComponent element?". If yes, "Why they are going to be useful if the user will instead configure this addon from the Web UI/Dashboard?"

**Comment by humitos** [comment]
We deployed the admin page for the addons. We should be able to start adding more per-addon settings now 🎉

**Comment by Alan-eMartin** [comment]
Hi @humitos,

I was wondering if there's a way to completely disable the hotkeys? My team uses a custom search modal/command palette widget for global search across multiple apps, and it’s encapsulated in a shadow DOM. The hotkeys interfere with its functionality—specifically, when a user types "d" or "/", it's not blocked.

Thanks for your help!

**Comment by humitos** [comment]
> I was wondering if there's a way to completely disable the search dialog and other hotkeys?

Yes. You can go to your addons project's configuration and de-activate hotkeys. Follow https://docs.readthedocs.io/en/stable/addons.html#configuring-read-the-docs-addons
